addons/l10n_se/__manifest__.py

```diff
@@ -1,21 +1,41 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Copyright (C) 2018 Linserv Aktiebolag, Sweden (<https://www.linserv.se>).
+
```

Comment:
Please remove copyright notice. You are not the original creator of this account table and can therefor not claim copyright on it.

Reply:
It could be removed. Even that the file is completly different.